pkg/ccloudv2/networking.go

```diff
@@ -36,3 +36,33 @@ func (c *Client) UpdateNetwork(envId, id string, updateReq networkingv1.Networki
 	resp, httpResp, err := c.NetworkingClient.NetworksNetworkingV1Api.UpdateNetworkingV1Network(c.networkingApiContext(), id).NetworkingV1NetworkUpdate(updateReq).Execute()
 	return resp, errors.CatchCCloudV2Error(err, httpResp)
 }
+
+func (c *Client) ListNetworks(envId string) ([]networkingv1.NetworkingV1Network, error) {
+	var list []networkingv1.NetworkingV1Network
+
+	done := false
+	pageToken := ""
+	for !done {
+		page, err := c.executeListNetworks(envId, pageToken)
+		if err != nil {
+			return nil, err
+		}
+		list = append(list, page.GetData()...)
+
+		pageToken, done, err = extractNextPageToken(page.GetMetadata().Next)
+		if err != nil {
+			return nil, err
+		}
+	}
+	return list, nil
+}
+
+func (c *Client) executeListNetworks(envId, pageToken string) (networkingv1.NetworkingV1NetworkList, error) {
```

Comment:
Nit: I try to match the parameter names with the setter function names.
```suggestion
func (c *Client) executeListNetworks(environment, pageToken string) (networkingv1.NetworkingV1NetworkList, error) {
```

Reply:
Should I change the parameter names for the functions above to keep them consistent as well?

Reply:
Yes please!

Reply:
updated!